packages/editor/src/lib/editor/Editor.ts

```diff
@@ -7349,6 +7349,49 @@ export class Editor extends EventEmitter<TLEventMap> {
 		return this
 	}
 
+	/**
+	 * Fit a frame to its content.
+	 *
+	 * @param id - Id of the frame you wish to fit to content.
+	 *
+	 * @public
+	 */
+	fitFrameToContent(id: TLShapeId): this {
```

Comment:
i think frames are not a core shape (only groups for now) so this helper should live in the `tldraw` package rather than the `editor` package.